### DISCUSSION TITLE
Can't login to App Store in VM

### DISCUSSION THREAD

**Discussion by ustndagsemih** [cirruslabs/tart#721] [Q&A]
Hello,

I created a VM with sonoma-base image. When I try to login to the App Store, it says "Verification Failed". However I can login through browser in the same VM. Through "App Store" or "Settings > Sign In" it doesn't work. Are there any limitations on that? Does anyone have the same issue?

**Comment by fkorotkov** [comment]
Unfortunately using Apple Account inside Virtual Machines is not supported. It's a restriction made by Apple.
